config/config.yml

```diff
@@ -16,6 +16,11 @@ framework:
         enabled: true
     router:
         resource: "%kernel.project_dir%/config/routing.yml"
+    asset_mapper:
```

Comment:
Add the build path to `.gitignore` and `.dockerignore`, maybe also exclude it from the volume mount in dev.